### ISSUE TITLE
Inconsistent stockLocationLine quantities

### ISSUE THREAD

**Issue by beuss** [axelor/axelor-business-suite#2071]
I do not know how I ended up in this situation but if stockLocationLine quantities are inconsistent (because of late Stock module activation? because of exception thrown at wrong moment?), there is no way to go out of this situation.
Application should provide a batch allowing to recompute some or all cached quantities.

**Comment by gdu-axelor** [comment]
Yes it can be a good things. In the same way we have done a batch to recompute the partner balances using account move lines.

**Comment by beuss** [comment]
OK, I can take over this one (if you can assign it), I'm doing it for the prestashop addon anyway :)
